hw/dv/sv/alert_esc_agent/doc/index.md

```diff
@@ -1,4 +1,6 @@
-# ALERT_ESC Agent
+---
```

Comment:
In general, is it mandatory to always create a separate `doc` directory? I wonder if `index.md` can live alongside the other sources.